src/betterproto/lib/pydantic/google/protobuf/__init__.py

```diff
@@ -1,9 +1,16 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # sources: google/protobuf/any.proto, google/protobuf/api.proto, google/protobuf/descriptor.proto, google/protobuf/duration.proto, google/protobuf/empty.proto, google/protobuf/field_mask.proto, google/protobuf/source_context.proto, google/protobuf/struct.proto, google/protobuf/timestamp.proto, google/protobuf/type.proto, google/protobuf/wrappers.proto
 # plugin: python-betterproto
-
+# This file has been @generated
```

Comment:
@Gobot1234 I thought you specifically said you didn't want this @generated label on these files during this PR: https://github.com/danielgtaylor/python-betterproto/pull/568.

I'm opening a PR for a CHANGELOG and bump in version, and wondering if you think this is blocking.

Reply:
Ah yep good catch, would you mind removing it in your PR so it's easier to merge?